cypress/components/button-minor/button-minor.test.js

```diff
@@ -158,69 +158,45 @@ context("Test for Button Minor component", () => {
           </ButtonMinor>
         );
 
-        buttonDataComponent().should("have.css", cssValue, assertion);
+        buttonMinorComponent().should("have.css", cssValue, assertion);
       }
     );
 
-    it("should check Button Minor is disabled", () => {
-      CypressMountWithProviders(<ButtonMinorDifferentTypes disabled />);
+    it.each(indexes)(
+      "should check Button Minor is disabled for %s button",
```

Comment:
This will output as follows when you run the tests:

```
should check Button Minor is disabled for 0 button
```

I think it might be worth updating the descriptions of the tests using the `indexes` array so it's a little clearer what's going on, maybe something like this:

```
should check Button Minor is disabled for the button at index %s
```